invenio_records_resources/resources/record_response.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-Records-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Schemas for parameter parsing."""
+
+
+from flask_resources.responses import Response
+from werkzeug.http import quote_etag
+
+from ..pagination import Pagination
+
+DEFAULT_RESULTS_PER_PAGE = 25
+DEFAULT_MAX_RESULTS = 10000
+
+
+class RecordResponse(Response):
```

Comment:
Question: We need this because we haven't yet made the header serializers in Flask-Resources right?

I would add a note in the text here then, to indicate that ideally, this file should disappear.